obs/obs-service-impl/src/main/java/com/adorsys/webank/obs/serviceimpl/OtpServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.adorsys.webank.obs.serviceimpl;
+
+import com.adorsys.webank.obs.service.OtpServiceApi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OtpServiceImpl implements OtpServiceApi {
+    private static final Logger logger = LoggerFactory.getLogger(OtpServiceImpl.class);
+
+    @Override
+    public String receiveOtp(String otp) {
+        if (otp == null || otp.length() != 6) {
+            logger.error("Invalid OTP format received.");
+            throw new IllegalArgumentException("Invalid OTP format");
```

Comment:
Please define a checked exception and make visible on the contract of this component.